### PR TITLE
Bump packages version to latest 

### PR DESCRIPTION
**Pull request by BiBamba**

### CODE REVIEW

File: jobs/core/templates/bin/vault [hunk 1]
```diff
@@ -20,13 +20,19 @@ case $1 in
...
     fi
 
-    if chpst -u vcap:vcap /var/vcap/packages/vault/bin/canimlock; then
-      sed -i -e 's/^disable_mlock/#disable_mlock/' \
-        $JOB_DIR/config/vault.conf
-    else
-      sed -i -e 's/^#disable_mlock/disable_mlock/' \
-        $JOB_DIR/config/vault.conf
-    fi
+    #if chpst -u vcap:vcap /var/vcap/packages/vault/bin/canimlock; then
```
Comment (tpoland) [inline jobs/core/templates/bin/vault:23]: We should remove the commented-out blocks of code and allow versioning to be our historical reference.

File: packages/vault/packaging [hunk 2]
```diff
@@ -3,12 +3,12 @@ set -eux
...
 mkdir $BOSH_INSTALL_TARGET/bin
 
-pushd mlock
-  make maybe
-  cp maybe $BOSH_INSTALL_TARGET/bin/canimlock
-popd
+#pushd mlock
+#  make maybe
+#  cp maybe $BOSH_INSTALL_TARGET/bin/canimlock
+#popd
```
Comment (tpoland) [inline packages/vault/packaging:9]: Let's remove the commented-out blocks

File: packages/vault/spec [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
 name: vault
 files:
-- vault/vault_0.9.0_linux_amd64.zip
-- mlock/maybe.c
+- vault/vault_1.11.4_linux_amd64.zip
+#- mlock/maybe.c
```
Comment (tpoland) [inline packages/vault/spec:5]: Remove commented out block.

**Review by tpoland — CHANGES_REQUESTED** [review]
A couple of changes are necessary to remove commented-out blocks of code.

File: jobs/core/templates/bin/vault (hunk 1 above)
Comment (BiBamba) [reply jobs/core/templates/bin/vault]: I removed the commented lines.

**Review by tpoland — APPROVED** [review]
LGTM

**Pull request merged**